Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -507,6 +507,10 @@ def genGrid(data1, data2, gm, deep=True, grid=None, geo=None):
         vg.SetPoints(geopts)
     else:
         vg = grid
+    # Add a GlobalIds array to keep track of cell ids throughout the pipeline
+    globalIds = numpy_to_vtk_wrapper(numpy.arange(0, vg.GetNumberOfCells()), deep=True)
```

Comment:
What do you mean 'by default'? In the uvcdat case we have threshold + different mapper for different scalar ranges so the ids are not the dataset ids. If you have only the dataset in the pipeline I think only the polydata will not change ids. Maybe there are others but I would not bet on it :-)